package.json

```diff
@@ -25,5 +25,8 @@
   },
   "dependencies": {
     "less": "^3.9.0"
-  }
+  },
+  "files": [
+    "index.js"
+  ]
```

Comment:
README & LICENSE are exclude. Is it okay ?
I checked other hexo libraries (not all), they are include README & LICENSE.